### PR TITLE
BETA: Register nino-os.is-a.dev

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: domains/nino-os.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Robotic-Germany",
+        "email": "rocker.lukas.wrld.16@gmail.com"
+    },
+    "record": {
+        "URL": "nino-os.dev"
```
Comment (DEV-DIBSTER) [inline domains/nino-os.json:7]: Unable to access site.

File: domains/nino_os.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Robotic-Germany",
+        "email": "rocker.lukas.wrld.16@gmail.com"
+    },
+    "record": {
+        "URL": "https://nino-os.great-site.net/"
```
Comment (DEV-DIBSTER) [inline domains/nino_os.json:7]: Unable to access site.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/nino-os.json (hunk 1 above)
Comment (Robotic-Germany) [reply domains/nino-os.json:7]: Can you please delete this website/file, I changed my mind and now don't want it anymore :)

File: domains/nino_os.json (hunk 2 above)
Comment (Robotic-Germany) [reply domains/nino_os.json:7]: Can you please delete this website/file, I changed my mind and now don't want it anymore :)

**Pull request closed without merging**